end-to-end-tests/cypress/support/e2e.js

```diff
@@ -19,18 +19,20 @@ import './commands'
 // Alternatively you can use CommonJS syntax:
 // require('./commands')
 
-Cypress.on('uncaught:exception', (err, runnable) => {
+Cypress.on('uncaught:exception', (err) => {
   // monaco-editor sometimes throws errors when reloading
   // and we don't want to fail the test so we return false
   if (err.message.includes("Failed to execute 'importScripts' on 'WorkerGlobalScope'")) {
     return false
   }
+
   if (err.message.includes("Cannot read properties of null (reading 'getText')")) {
     return false
   }
+  
   // TODO: Find a fix where we don't need to avoid this exception
   // https://github.com/EasyDynamics/oscal-editor-deployment/issues/121
-  if (err.message.includes("ResizeObserver loop limit exceeded")) {
+  if (err.message.includes("ResizeObserver loop")) {
```

Comment:
Looking at the cypress output, the error message did not always match "ResizeObserver loop limit exceeded". It always has "ResizeObserver loop", so I just shorted the `includes`.